### ISSUE TITLE
example

### ISSUE THREAD

**Issue by gedw99** [pbegle/singer-go#1]
singer.io looks really nice. never knew there was an open agreed specification

Was wondering if you have any info about how to use it with singer as i am also a goang coder.

Also this reminds me of benthos ETL golang system..

**Comment by pbegle** [comment]
Yeah, I’ll be adding examples. Still actively working on this repo so everything is still work in progress but I will definitely add some examples on how to actually use it in practice.

Thank you,

Peter Begle
